services/core-api/app/api/now_applications/models/state_of_land.py

```diff
@@ -15,10 +15,15 @@ class StateOfLand(Base):
         db.Integer, db.ForeignKey('now_application.now_application_id'), primary_key=True)
     now_application = db.relationship('NOWApplication')
 
-    has_community_water_shed = db.Column(db.Boolean, nullable=False, server_default=FetchedValue())
-    has_archaeology_sites_affected = db.Column(
-        db.Boolean, nullable=False, server_default=FetchedValue())
-
+    has_community_water_shed = db.Column(db.Boolean, nullable=True)
+    has_archaeology_sites_affected = db.Column(db.Boolean, nullable=True)
+
+    authorization_details = db.Column(db.String)
+    has_licence_of_occupation = db.Column(db.Boolean, nullable=True)
+    licence_of_occupation = db.Column(db.String)
+    file_number_of_app = db.Column(db.String)
+    applied_for_license_of_occupation = db.Column(db.Boolean, nullable=True)
```

Comment:
This "license" should be spelled as "licence" because it's a noun here

Reply:
updated.